Modules/_io/bufferedio.c

```diff
@@ -1767,6 +1778,27 @@ _bufferedwriter_reset_buf(buffered *self)
     self->write_end = -1;
 }
 
+static void
+_bufferedwriter_set_append(buffered *self)
+{
+    PyObject *mode;
+
+    mode = _PyObject_GetAttrId(self->raw, &PyId_mode);
+    if (mode != NULL) {
+        /* Raw fileobj has no mode string so as far as we can know it has
+           normal write behavior */
+        if (PyUnicode_FindChar(mode, 'a', 0, PyUnicode_GET_LENGTH(mode), 1) != -1) {
+            self->appending = 1;
+        } else {
```

Comment:
```suggestion
        }
        else {
```
The `}` should go on its own line (according to PEP 7).